db/seeds.rb

```diff
@@ -1,7 +1,41 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'faker'
+
+interrogatives = [ "Who", "What", "When", "Where", "Why", "Which", "How" ]
+nouns = []
+verbs = []
+choices = []
+20.times { nouns << Faker::Hacker.noun }
+20.times { verbs << Faker::Hacker.verb }
+50.times { choices << Faker::Commerce.product_name }
+
+5.times do
+  user_attributes = {
+  email: "#{Faker::Internet.email}",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.